src/core/Akka.Cluster.Tests.MultiNode/TransitionSpec.cs

```diff
@@ -61,7 +64,7 @@ private RoleName[] NonLeader(params RoleName[] roles)
 
         private MemberStatus MemberStatus(Address address)
         {
-            var status = ClusterView.Members.Concat(ClusterView.UnreachableMembers)
+            var status = ClusterView.Members.Union(ClusterView.UnreachableMembers)
```

Comment:
Used the `Union` method here to filter out duplicates and be consistent with the JVM.